_posts/2023-11-09-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,310 @@
+
+== Add Helm Configuration
+
+Lastly, we will be creating a helm release of our local WildFly image and deploying our application on it using a helm chart. Navigate to `charts` directory and open the `helm.yaml` file. 
+```
+cd /PATH/TO/ELYTRON/EXAMPLES/elytron-oidc-client-scope/charts
+```
+The contents of the file will be as follows: 
+```
+image: 
+  name: quay.io/<your-quay-username>/wildfly
+build:
+  enabled: false
+deploy:
+  env:
+    - name: OIDC_PROVIDER_URL
+      value: '<OIDC_PROVIDER_URL>'/auth/
+    - name: WILDFLY_OVERRIDING_ENV_VARS
+      value: "1"
+    - name: SUBSYSTEM_LOGGING_PATTERN_FORMATTER_COLOR_PATTERN__PATTERN
+      value: "[redhat-openshift] %-5p %s%e%n"
+```
+Replace <OIDC_PROVIDER_URL> to match the url for your RH-SSO instance on OpenShift and add a `/auth/` to the end of the
+URL as seen above. There are three fields that we have specified: image, build and deploy.
+
+Using the image tag, we are specifying that we want the pod to use the image of our application that we just built.
+Notice that we did not specify the tag. This is because when the helm chart uploads, the `latest` tag is appended to it
+by default, which is why we tagged our application image as that. If you want to use a different tag, then you can edit
+this by clicking on the 3 dots at the bottom of the pod, click on *Edit Deployment* and changing it under *Image Name*.
+
+We have also added some environment variables. Notice the `OIDC_PROVIDER_URL`. The value of this will be the url for
+your RH-SSO pod, which can be found by clicking on the `Open URL` button on the pod. So, copy that link and paste it as
+the `value`. The `OIDC_PROVIDER_URL` variable is being used by the `oidc.json` file inside the `WEB-INF` directory when
+specifying the `provider-url`.
+
+Change the image name on the `charts/helm.yaml` file to include your quay username instead of <your-quay-username>.
+
+== Deploy the Example Application to WildFly on OpenShift
+Use the following commands to create a cloud instance for your webapp using the wildfly helm chart: 
+```
+helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+helm install simple-webapp-oidc -f charts/helm.yaml wildfly/wildfly
+```
+After running the commands above, you should expect to see the following output:
+```
+Waited for 1.068205845s due to client-side throttling, not priority and fairness,
+request: GET:https://api.sandbox-m4.g2pi.p1.openshiftapps.com:6443/apis/triggers.tekton.dev/v1beta1?timeout=32s
+NAME: simple-webapp-oidc
+LAST DEPLOYED: <current date and time>
+NAMESPACE: <your namespace>
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+To follow the deployment of your application, run:
+
+$ oc get deployment simple-webapp-oidc -w
+```
+Click on the three dots beside the `simple-webapp-oidc` pod's name and click on *Edit Deployment*.
```

Comment:
just a total minor, we can mention that the deployment will faill and to make it succeed, we have to click on the three dots etc.

Reply:
fixed! 
Thanks @Skyllarr for trying it out and for your feedback!